adapter/sql/util.go

```diff
@@ -23,12 +23,21 @@ type fieldCacheKey struct {
 	escape string
 }
 
+type aliasField struct {
+	source string
+	alias  string
+}
+
 // Escape field or table name.
 func Escape(config Config, field string) string {
 	if config.EscapeChar == "" || field == "*" {
 		return field
 	}
 
+	if v, ok := isAliasField(field); ok {
```

Comment:
I think we should cache the result as well by moving this to line 46

Reply:
That's ok, will update soon